web/src/features/charts/BreakdownChart.tsx

```diff
@@ -75,7 +75,7 @@ function BreakdownChart({
       <div className="relative">
         {isBreakdownGraphOverlayEnabled && (
           <div className="absolute top-0 h-full w-full">
-            <div className=" h-full w-full bg-white opacity-50 dark:bg-gray-800" />
+            <div className=" h-full w-full scale-x-150 bg-white opacity-50 dark:bg-gray-800" />
```

Comment:
I think this is a bit of a hack but I couldn't really figure out a better way to make the child div be wider than its parent.